src/swapnaWJun20/SearchItemId.java

```diff
@@ -0,0 +1,27 @@
+package swapnaWJun20;
+
+public class SearchItemId {
+
+	void findItemID(String input) {
+		System.out.println("\nInput : " + input);
+		if (input.contains("itemId")) {
+			int itemIndex = input.lastIndexOf("itemId=");
+			int slashIndex = input.lastIndexOf("/");
+			String output = input.substring(itemIndex, slashIndex).replace("itemId=", "");
+			if (output.contains("empid")) {
+				output = output.substring(0, output.lastIndexOf("&"));
+			}
```

Comment:
if (input.contains("itemId")) {
			int item = input.indexOf("itemId");
			String sub = input.substring(item);
			String s = sub.split("=")[1].split("&")[0].split("/")[0];
			System.out.println("ItemID is " + s);
		} else {
			System.out.println("No itemID present.");
		}
Above is optimize solution.

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.